src/resources/js/ticket-delete-alert.js

```diff
@@ -0,0 +1,6 @@
+(function( window, $ ) {
+	'use strict';
+	$('.ticket_delete').click( function(){
+		alert( ticket_notices.alert );
```

Comment:
This is my lack of js experience showing.  Alert allows the user to confirm before the action takes place.  I'm not able to make other methods work...I've tried using `prevent.default`.  I'll do a bit more tweaking to see if I can come up with a modal type solution.